Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Select.hs

```diff
@@ -66,25 +68,30 @@ handler merchant sReq estimate = do
   searchReq <- QSR.findById estimate.requestId >>= fromMaybeM (SearchRequestNotFound estimate.requestId.getId)
   farePolicy <- getFarePolicy merchantId estimate.vehicleVariant searchReq.area
 
-  searchTry <- createNewSearchTry farePolicy searchReq
-  driverPoolConfig <- getDriverPoolConfig merchantId searchReq.estimatedDistance
-  let inTime = fromIntegral driverPoolConfig.singleBatchProcessTime
+  Redis.whenWithLockRedis (newSearchTryLockKey searchReq.id.getId) 60 $ do
```

Comment:
It doesn't throw any error, so frontend won't know if select request ended successfully